admin-dev/themes/default/template/controllers/orders/form.tpl

```diff
@@ -479,6 +479,11 @@
     var customer_search = $customer_search_input.val();
     var customer_search_url = $customer_search_input.data('customers-search-url');
 
+    // id_customer parameter is reuqired to generate url
+    // in this case id_customer is dynamic
+    // so 0 is used as placeholder and later replaced with actual id_customer
+    var customer_view_url_template = '{$link->getAdminLink('AdminCustomers', true, [], ['viewcustomer' => 1, 'liteDisplaying' => 1, 'id_customer' => 0])}';
```

Comment:
The deeper into the forest, the more trees around... :sweat:

Reply:
You become better at poetry. They say great poets have a broken heart 💔 I think with this migration project you are on the right path :trollface:

Reply:
I don't know if I should cry or laugh. :rofl: